### DISCUSSION TITLE
Aim integration?

### DISCUSSION THREAD

**Discussion by davzaman** [guildai/guildai#464] [Ideas]
I know guild uses tensorboard for its tracking and comparison but I've recently been keeping my eye on [aim](https://github.com/aimhubio/aim) which is a bit young but looks very promising. It's nicer to look at and easier to use than tensorboard. Their claim is it's faster when you have a lot of runs, I haven't used it enough to have a one-off opinion about performance, but I think it's worth looking into! While tensorboard has a lot of plugins I like the way aim allows you to slice and dice and do some basic visualization on the go right in their UI.

**Comment by gar1t** [comment]
Hi! This is a really good idea. Guild is wired for integration with tools like this (ala TensorBoard) and I agree that this project provides a lot of advantages over TensorBoard. It a sensible factoring of runs and artifact/metric views to those runs!
